image/verify/src/verifier.rs

```diff
@@ -110,10 +110,30 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
         let image_info = okref(&image_info)?;
 
         // Verify FMC
-        let (fmc_info, fmc_log_info) = self.verify_fmc(image_info.fmc, reason)?;
+        let fmc_info = self.verify_fmc(image_info.fmc, reason)?;
 
         // Verify Runtime
-        let (runtime_info, rt_log_info) = self.verify_runtime(image_info.runtime)?;
+        let runtime_info = self.verify_runtime(image_info.runtime)?;
+
+        // Verify SVN
+        if self.svn_check_required() {
+            if image_info.fmc.svn != image_info.runtime.svn {
+                Err(CaliptraError::IMAGE_VERIFIER_ERR_RUNTIME_FMC_SVNS_UNEQUAL)?;
```

Comment:
Not sure what to make of this. Shouldn't we just have one SVN now?

Reply:
I was trying to avoid changing the image layout, for backwards compatibility. The FMC and RT FW images within the bundle each separately carry an SVN term. Tooling now writes the same value down when generating an image, and ROM ensures that the two are equal.

Have added a note to that effect.